### ISSUE TITLE
Does DIA-NN incorporate knowledge that peptides at different charge states should have same/similar RTs?

### ISSUE THREAD

**Issue by gsaxena888** [vdemichev/DiaNN#617]
Does DIA-NN incorporate knowledge that peptides at different charge states should have same/similar RTs? (That is, does it give some sort of "boost" in some sort of scoring logic when identified peptides appear at different charge states but identical (or say nearly identical, i.e., < 7 second difference) in retention time? If so, it would be a way of recovering "hard to identify" peptides in one charge state by using the fact that the same peptide exists in a different charge state **but** at the same (or similar enough) retention time) Thoughts?

**Comment by vdemichev** [comment]
Yes, if you use --peak-translation. But the effect of this is minor if any.

Best,
Vadim
